src/count.js

```diff
@@ -20,6 +20,9 @@ function count(ast) {
     case 'keyframes':
     case 'import':
     case 'supports':
+    case 'charset':
+    case 'namespace':
+    case 'page':
```

Comment:
Yep, I was wondering that as well, having a switch case of implicit at-rules, is not very scalable and future-proof (eg, if for some reason W3C introduced 100 new at-rules in the future), but then again it's just me looking at these few lines of code, really short-sighted view of the bless plugin, since I don't know what the switch case was for, the parsing and how it affects/importance to the plugin as a whole.

But ye I agree, ideally it would be nice to have a more general, flexible fix instead, for the meantime the main purpose of this change is to fix the bug at hand. I'm using a plugin that depends on another plugin that depends on this (inception...lol), so it's hard for me to manually rollback to `Bless v3` in auto deploy.